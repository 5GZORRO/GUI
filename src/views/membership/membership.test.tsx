// @ts-nocheck
import React from 'react'
import TestRenderer from 'react-test-renderer'
import FormGenerate from './index'

describe('Membership', function () {
  it('Snapshot ', () => {
    const tree = TestRenderer.create(
        <FormGenerate />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
