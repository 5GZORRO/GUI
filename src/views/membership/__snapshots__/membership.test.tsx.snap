// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Membership Snapshot  1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-sm-12"
    >
      <form
        className=""
        onSubmit={[Function]}
      >
        <div
          className="form-group"
        >
          <label
            className=""
            htmlFor="nf-email"
          >
            Text
          </label>
          <input
            className="form-control"
            onBlur={[Function]}
            onChange={[Function]}
            selected={null}
            type="text"
          />
        </div>
        <div
          className="form-group"
        >
          <label
            className=""
            htmlFor="Type"
          >
            Type
          </label>
        </div>
        <button
          className="btn btn-primary"
          onClick={[Function]}
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
  </div>
</div>
`;
